### ISSUE TITLE
record my result on different model

### ISSUE THREAD

**Issue by wurongyuan** [kuangliu/pytorch-cifar#112]
- ubuntu16.04   
- pytorch1.4   
- cpu: 3700x   
- gpu: 2070s

mobilenetv1   ===>  92.350
=
### settings: 

lr = 0.1     batch_size = 128    epoch = 350
optimizer = optim.SGD(net.parameters(), lr=args.lr, momentum=0.9, weight_decay=4e-5)
scheduler = optim.lr_scheduler.MultiStepLR(optimizer, milestones=[150, 250], gamma=0.1)

### best result  :+1: 

```
Epoch: 344
 391/391 [================================================================================>]  Step: 15ms | Tot: 9s285ms | Loss: 0.002 | Acc: 99.942% (49971/50000)
 79/79 [================================================================================>]  Step: 2ms | Tot: 593ms | Loss: 0.461 | Acc: 92.350% (9235/10000)
Saving..
```

mobilenetv2   ===>  94.040
=
### settings: 

lr = 0.1     batch_size = 128    epoch = 350
optimizer = optim.SGD(net.parameters(), lr=args.lr, momentum=0.9, weight_decay=4e-5)
scheduler = optim.lr_scheduler.MultiStepLR(optimizer, milestones=[150, 250], gamma=0.1)

### best result  :

```
Epoch: 291
 391/391 [================================================================================>]  Step: 50ms | Tot: 30s938ms | Loss: 0.001 | Acc: 99.992% (49996/50000)
 79/79 [================================================================================>]  Step: 5ms | Tot: 1s651ms | Loss: 0.326 | Acc: 94.040% (9404/10000)
Saving..
```
